src/test/java/uk/ac/ebi/eva/pipeline/jobs/steps/VariantLoaderStepTest.java

```diff
@@ -99,7 +100,13 @@ public void loaderStepShouldLoadAllVariants() throws Exception {
                 VariantSource.Aggregation.NONE));
```

Comment:
They are still on Job Options and Cristina disagreed to remove it completely until no one requires them.